README.md

```diff
@@ -97,14 +97,37 @@ $ balena run -it -v ${appid}_resin-data:/mnt alpine \
 
 ## Networking
 ### Shared physical device
-First, create a bridge, and assign the physical interface as a slave. This can be done on the host OS using nmcli:
+
+Check your network before you wreck your network:
+
+Find the ethernet name for example something like this eno1 or enp89s0 
+`` 
+$ ip a
+``
+
+First, create a bridge on thee ethernet port preferably(wifi will require lan and dhcp server), and assign the physical interface as a slave. This can be done on the host OS using nmcli:
```

Comment:
This needs reworked.
```suggestion
For technical reasons, a wired ethernet interface will work with minimal complication. Wireless interfaces cannot be attached to a Linux host bridge. It's possible to setup another subnet, and use NAT to forward traffic over the wireless interface, but that is beyond the scope of this readme.

Create a bridge, and add the physical interface as a slave. This can be done on the host OS using nmcli: (replace eno1 here with your interface)
```